contracts/VeJoeStaking.sol

```diff
@@ -87,17 +93,26 @@ contract VeJoeStaking is Initializable, OwnableUpgradeable {
         uint256 _boostedDuration,
         uint256 _maxCap
     ) public initializer {
+        UPPER_LIMIT_BOOSTED_DURATION = 365 days;
```

Comment:
Optional, you can define this just before the `require` statements where they are used, saving gas on failed txs.